src/components/top_bar/top_bar_tool_store.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from "@odoo/owl";
+
+export class TopBarToolStore {
```

Comment:
```suggestion
/**
 * This store is used to manage the dropdown that is currently opened after clicking an item on the toolbar. It can only have one displayed at a time.
 *
 */
export class TopBarToolStore {
```